backend/controller/usercontroller.js

```diff
@@ -68,12 +74,19 @@ const getUsers = asyncHandler(async (req, res, next) => {
     };
   }
 
+  result.totalPages = Math.floor(totalUsers / LIMIT);
   result.users = await userModel
     .find(query)
     .skip(startIndex)
     .limit(LIMIT)
     .sort({ createdAt: 1 });
-  return res.status(200).json({ success: true, data: result });
+  return res.status(200).json({
+    status: 200,
+    success: true,
+    message:
+      result.users.length > 0 ? "Fetch users successfully" : "Users not found",
```

Comment:
Instead of Users not found say No user found

Reply:
done